code/modules/mining/machine_processing.dm

```diff
@@ -181,41 +184,47 @@
 		unload_mineral(O)
 	else
 		materials.insert_item(O)
-		if(mineral_machine)
-			mineral_machine.updateUsrDialog()
-
-/obj/machinery/mineral/processing_unit/proc/get_machine_data()
-	var/dat = "<b>Smelter control console</b><br><br>"
-	for(var/datum/material/all_materials as anything in materials.materials)
-		var/amount = materials.materials[all_materials]
-		dat += "<span class=\"res_name\">[all_materials.name]: </span>[amount] cm&sup3;"
```

Comment:
The UI you show doesn't give the amount of material is in the machine, can you add that since it's pretty important to know what you should be smelting.

Reply:
Oh, my bad! I'll add it.